src/internal/item-container/index.tsx

```diff
@@ -399,13 +440,52 @@ function ItemContainerComponent(
     }
   }
 
-  const dragHandleRef = useRef<HTMLButtonElement>(null);
+  const dragHandleRef = useRef<HTMLDivElement>(null);
   useImperativeHandle(ref, () => ({
-    focusDragHandle: () => dragHandleRef.current?.focus(),
+    focusDragHandle: () => {
+      return dragHandleRef.current?.focus();
+    },
   }));
 
+  const hookProps: UseInternalDragHandleInteractionStateProps<HandleOperation> = {
+    onDndStartAction: (event, handleOperation) => {
+      if (handleOperation === "drag") {
+        onDragHandleDndTransitionStart(event);
+        return;
+      }
+      onResizeHandlePointerDown(event);
+    },
+    onDndActiveAction: (event) => {
+      onHandleDndTransitionActive(event);
+    },
+    onDndEndAction: () => {
+      if (transition) {
+        draggableApi.submitTransition();
+      }
+    },
+    onUapActionStartAction: (handleOperation) => {
+      handleKeyboardTransition(handleOperation!);
```

Comment:
If we reuse `handleKeyboardTransition` for keyboard and uap - would it make sense to call it e.g. `handleIncrementalTransition` (because the user moves/resizes items in small increments)?

Reply:
Great suggestion, I think `handleIncrementalTransition` is a more appropriate name. Changing it.